extensions/screenshot-full-page-in-browser/package.json

```
@@ -0,0 +1,44 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "screenshot-full-page-in-browser",
+  "title": "Screenshot  full page in browser",
+  "description": "This extension allows you to quickly make a screenshot of the entire page of the site and save it to your desktop just by writing the address of the site.",
+  "icon": "command-icon.png",
+  "author": "Coun1er",
+  "categories": [
+    "Web",
+    "Developer Tools",
+    "Productivity"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "screenshot-full-page",
+      "title": "Screenshot full page in browser",
```

```suggestion
      "title": "Capture Fullpage Screenshot of Website",
```
Could this be a better title?